src/Console/Command/DescribeCommand.php

```diff
@@ -204,24 +228,119 @@ private function arrayToText(array $data)
         );
     }
 
+    /**
+     * @param bool   $isDecoratedOutput
+     * @param string $diff
+     *
+     * @return string
+     */
     private function prepareDiff($isDecoratedOutput, $diff)
     {
-        $template = "<comment>   ---------- begin diff ----------</comment>\n%s\n<comment>   ----------- end diff -----------</comment>";
-
-        $diff = implode(
-            "\n",
-            array_map(
-                function ($string) {
-                    $string = preg_replace('/^(\+.*)/', '<fg=green>\1</>', $string);
-                    $string = preg_replace('/^(\-.*)/', '<fg=red>\1</>', $string);
-                    $string = preg_replace('/^(@.*)/', '<fg=cyan>\1</>', $string);
-
-                    return '   '.$string;
-                },
-                preg_split("#\n\r|\n#", OutputFormatter::escape(rtrim($diff)))
-            )
-        );
+        if ($isDecoratedOutput) {
+            $template = "<comment>   ---------- begin diff ----------</comment>\n%s\n<comment>   ----------- end diff -----------</comment>";
+            $diff = implode(
+                PHP_EOL,
+                array_map(
+                    function ($string) {
+                        $string = preg_replace('/^(\+.*)/', '<fg=green>\1</>', $string);
+                        $string = preg_replace('/^(\-.*)/', '<fg=red>\1</>', $string);
+                        $string = preg_replace('/^(@.*)/', '<fg=cyan>\1</>', $string);
+
+                        return '   '.$string;
+                    },
+                    preg_split("#\n\r|\n#", OutputFormatter::escape(rtrim($diff)))
+                )
+            );
+        } else {
+            $template = '      ---------- begin diff ----------%s      ----------- end diff -----------';
```

Comment:
no no no no no, I initially remove that checking `isDecoratedOutput`. it's not needed and it looks worse when you change the template.